src/Redistribution.sol

```diff
@@ -191,7 +214,7 @@ contract Redistribution is AccessControl, Pausable {
      */
     function currentPhaseReveal() public view returns (bool) {
         uint256 number = block.number % roundLength;
-        if ( number >= roundLength / 4 && number < roundLength / 2 ) {
+        if (number >= roundLength / 4 && number < roundLength / 2) {
```

Comment:
let's make sure commits do not have formatting changes to improve readability 🤓

Reply:
Sorry for that. Usually, I do formatting changes in one commit and actual changes in another so it's easier to track what is what.